### PR TITLE
Added note about needing 75MiB per host

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1538944

@sdodson PTAL. Is this placement okay, or would you expect to see this noted elsewhere? Thanks!

### CODE REVIEW

**Comment by sdodson** [comment]
/lgtm
thanks!

**Comment by ahardin-rh** [comment]
@openshift/team-documentation PTAL

File: install_config/cluster_metrics.adoc [hunk 1]
```diff
@@ -539,6 +539,12 @@ Because deploying and configuring all the metric components is handled with
 The following examples show you how to deploy metrics with and without
 persistent storage using the default parameters.
 
+[IMPORTANT]
+====
+The host where you are running Ansible playbook must have at least 75MiB of free
```
Comment (kalexand-rh) [inline install_config/cluster_metrics.adoc]: s/where you are running Ansible playbook/that you run the Ansible playbook on

**Comment by kalexand-rh** [comment]
@ahardin-rh, one pick.

**Comment by ahardin-rh** [comment]
Thanks! Updates complete and QE comments from the BZ are also addressed.

**Pull request merged**